src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -24,6 +24,7 @@ Example:
 
 """
 type IterativeSolver<: AbstractSolver
+   AA   # matrix being solved
```

Comment:
My two cents: 

I've been doing some major refactoring to MaxwellTime in preparation for making it public (see the current partially completed code on the interface-refactoring branch of https://github.com/Pbellive/MaxwellTime). I've added a field `storageLevel::Symbol` to the MaxwellTime param. This is a categorical option with possible values `:Factors`, `:Matrices`, and `:None`. In the case the `:Factors` is chosen and direct solvers are being used then factorizations are stored and matrices are discarded. When iterative solvers are used, the user may choose whether or not they wish to store the matrices.

This approach makes the code slightly uglier but it has the significant advantages of making it explicit to the user what's being stored, giving them control over the storage, not storing matrices unnecessarily, and keeping matrices out of `linearSolvers` objects. In time domain EM where several matrices factorizations are needed for a single forward problem this is a bigger deal than for elliptic pdes but the same approach could be applied generally.